### PR TITLE
Normalizes dc:type image.

### LINKED ISSUE #1631: Normalize image
For druid:tb744wy6365:
```
<extension displayLabel="geo">
  <rdf:RDF xmlns:gml="http://www.opengis.net/gml/3.2/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:gmd="http://www.isotc211.org/2005/gmd">
    <rdf:Description rdf:about="http://purl.stanford.edu/tb744wy6365">
      <dc:format>image/jpeg</dc:format>
      <dc:type>image</dc:type>
      <gmd:centerPoint>
        <gml:Point gml:id="ID">
          <gml:pos>31.201435 29.909773</gml:pos>
        </gml:Point>
      </gmd:centerPoint>
    </rdf:Description>
  </rdf:RDF>
</extension>
Best guess roundtripped node:
<extension displayLabel="geo">
  <rdf:RDF xmlns:gml="http://www.opengis.net/gml/3.2/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:gmd="http://www.isotc211.org/2005/gmd">
    <rdf:Description rdf:about="http://purl.stanford.edu/tb744wy6365">
      <dc:format>image/jpeg</dc:format>
      <dc:type>Image</dc:type>
      <gmd:centerPoint>
        <gml:Point gml:id="ID">
          <gml:pos>31.201435 29.909773</gml:pos>
        </gml:Point>
      </gmd:centerPoint>
    </rdf:Description>
  </rdf:RDF>
</extension>
```

"image" should be normalized to "Image".

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1631

## Why was this change made?
Nornalize bad data.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   87515 (87.515%)
  Different: 11819 (11.819%)
  To Cocina error:     32 (0.032%)
  To Fedora error:     24 (0.024%)
  Missing:     610 (0.61%)
```

After:
```
Status (n=100000):
  Success:   87515 (87.515%)
  Different: 11819 (11.819%)
  To Cocina error:     32 (0.032%)
  To Fedora error:     24 (0.024%)
  Missing:     610 (0.61%)
```

File: spec/services/cocina/mods_normalizer_spec.rb [hunk 1]
```diff
@@ -896,4 +896,45 @@
...
+                <dc:format>image/jpeg</dc:format>
+                <dc:type>image</dc:type>
+              </rdf:Description>
+            </rdf:RDF>
+          </extension>
+        </mods>
+      XML
+    end
+
+    it 'fix capitalization' do
```
Comment (ndushay) [inline spec/services/cocina/mods_normalizer_spec.rb:920]: how about - it uses a capital I ??  more descriptive?     Also, maybe example could have both capital and lowercase to show it comes out capital I in both cases?

**Review by ndushay — APPROVED** [review]
approving with a comment for you to consider.

File: spec/services/cocina/mods_normalizer_spec.rb (hunk 1 above)
Comment (justinlittman) [reply spec/services/cocina/mods_normalizer_spec.rb]: Both suggestions added.

**Pull request merged**